tests/Auth/AuthorizeMiddlewareTest.php

```diff
@@ -19,6 +19,8 @@
 use PHPUnit\Framework\TestCase;
 use stdClass;
 
+include_once 'Enums.php';
```

Comment:
Same as the other comment, I guess it is best to use an import instead of `include_once` to import the enum.

```php
use Illuminate\Tests\Auth\AbilitiesEnum;
```

Reply:
@rodrigopedra The same response as above ^_^